### PR TITLE
Fixing bug in mesh rasterizer

### PR DESCRIPTION
**Pull request by myla**
In the file [rasterizer.py](https://github.com/facebookresearch/pytorch3d/blob/de3a474d2b9e0f5d4a77c82106a4a8e7853a4e07/pytorch3d/renderer/mesh/rasterizer.py#L201) there are duplicate lines before the check if the `projection_transform` exists. These cause an exception in the case that a projection transform matrix is already provided. The corresponding lines should be (and are already) in the else case of the if-statement.

Removing these lines fixes the bug and produces the desired behaviour.

### CODE REVIEW

File: pytorch3d/renderer/mesh/rasterizer.py [hunk 1]
```diff
@@ -198,10 +198,6 @@ def transform(self, meshes_world, **kwargs) -> torch.Tensor:
         verts_view = cameras.get_world_to_view_transform(**kwargs).transform_points(
             verts_world, eps=eps
         )
-        # Call transform_points instead of explicitly composing transforms to handle
-        # the case, where camera class does not have a projection matrix form.
-        verts_proj = cameras.transform_points(verts_world, eps=eps)
-        to_ndc_transform = cameras.get_ndc_camera_transform(**kwargs)
```
Comment (bottler) [inline pytorch3d/renderer/mesh/rasterizer.py:204]: Did you mean to leave this line in and only remove the previous line? If so, then I think the fix is right.

Comment (myla) [reply pytorch3d/renderer/mesh/rasterizer.py:204]: Yes, sorry for that. I only meant to remove the line containing the camera.transform_points() and the comment, as it makes more sense in the else part.

Comment (myla) [reply pytorch3d/renderer/mesh/rasterizer.py:204]: I created a second commit in the branch to resolve the issue. Let me know if you want a new pull request.

Comment (myla) [reply pytorch3d/renderer/mesh/rasterizer.py:204]: Created a new pull request where the issue is fixed.

**Comment by myla** [comment]
Created an updated pull request, closing this.

**Comment by bottler** [comment]
> Created an updated pull request, closing this.

Thanks. It's #1421 .

**Pull request closed without merging**